portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,9 +32,48 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Kyle!</h1>");
+    int numComments = Integer.parseInt(getParameter(request, "num-comments", "5"));
```

Comment:
The defaultValue can be a private static member variable in this class. I noticed that you have different default values, so you can create different variables such as DEFAULT_COMMENTS_COUNT

You can check the style guide https://google.github.io/styleguide/javaguide.html to use the correct style.